test/e2e/specs/liquidation.spec.js

```diff
@@ -428,28 +428,28 @@ describe('Wallet App Test Cases', () => {
     it('should verify the value of startPrice from the CLI successfully', () => {
       const propertyName = 'book0.startPrice';
       const expectedValue = '9.99 IST/ATOM';
-
+      cy.wait(3000); // eslint-disable-line cypress/no-unnecessary-waiting
       cy.verifyAuctionData(propertyName, expectedValue);
```

Comment:
cant we just add this line inside cy.verifyAuctionData as well?

Reply:
We can do that. But it makes `verifyAuctionData` fail for some reason. I did not dig much into but perhaps I will then to `cy.wait().then()` in verifyAuctionData. Perhaps

Reply:
Yes, it is required there

Reply:
please update it as well